sql-plugin/src/main/scala/com/nvidia/spark/rapids/RmmRapidsRetryIterator.scala

```diff
@@ -835,51 +834,64 @@ object RmmRapidsRetryIterator extends Logging {
   /**
    * Log memory footprint when GPU OOM or CPU OOM happens.
    */
-
   val BOOKKEEP_MEMORY: Boolean =
     java.lang.Boolean.getBoolean("ai.rapids.memory.bookkeep")
   // track the callstack for each memory allocation, don't enable it unless really needed
   val BOOKKEEP_MEMORY_CALLSTACK: Boolean =
     java.lang.Boolean.getBoolean("ai.rapids.memory.bookkeep.callstack")
   // By default, only print first time to avoid too much log
-  val BOOKKEEP_MEMORY_PRINT_ALL: Boolean =
-    java.lang.Boolean.getBoolean("ai.rapids.memory.bookkeep.printall")
-  var bookkeepPrinted = false
+  val PRE_SPLIT_PRINT_ALL: Boolean =
```

Comment:
ai.rapids.memory.bookkeep.printall was not documented anywhere, and I believe I'm the only one using it. So it would be safe to rename it